mono/mini/interp/transform.c

```diff
@@ -4246,7 +4262,6 @@ generate (MonoMethod *method, MonoMethodHeader *header, InterpMethod *rtm, unsig
 			p = mono_interp_dis_mintop_len (p);
 		}
 	}
-	g_assert (td->max_stack_height <= (header->max_stack + 1));
```

Comment:
I feel like we should still have some sanity check here, like

```c
g_assert (td->max_stack_height <= header->max_stack * 3);
```

Reply:
I don't think we should assert, but I'm ok with a g_warning